addons/load-extension/userscript.js

```diff
@@ -0,0 +1,8 @@
+export default async function ({ addon, global, console }) {
+  // TODO: Add removing extensions.
```

Comment:
How could that work? The blocks would already be in the project...

Reply:
For non-used

Reply:
That's already something scratch does

Reply:
no, you would have to reload the page for it to be removed. This addon should be able to allow it to be removed without refreshing.

Reply:
Scratch waits for the refresh because otherwise you would have time to take blocks out.

Reply:
Wait do you mean manually removing them? Probably a different addon would be best.

Reply:
it fits the context enough. An addon that auto adds them, while also allowing removing.

Reply:
> it fits the context enough. An addon that auto adds them, while also allowing removing.

A single addon should not do multiple unrelated things.